quickstart/llmd-installer.sh

```diff
@@ -52,6 +53,7 @@ Options:
   -D, --download-model             Download the model to PVC from Hugging Face
   -t, --download-timeout           Timeout for model download job
   -k, --minikube                   Deploy on an existing minikube instance with hostPath storage
+  -g, --context                    Supply a specific Kubernete/OpenShift context
```

Comment:
Tiny request, can we change to `Supply a specific Kubernetes context`. Great feature!